server/src/rest/middleware.ts

```diff
@@ -31,6 +31,8 @@ export const errorHandler = (): ErrorRequestHandler =>
         return sendErrorResponse(res, statusCodes.STATUS_NOT_FOUND, error);
       case 'uniquenessFailed':
         return sendErrorResponse(res, statusCodes.STATUS_CONFLICT, error);
+      case 'nominationFailed':
+        return sendErrorResponse(res, statusCodes.STATUS_CONFLICT, error);
```

Comment:
I think we should return a `statusCodes.BAD_REQUEST` error for failed nominations.